templates/base.html

```diff
@@ -13,12 +13,12 @@
   </script>
 </head>
 <body>
-    {% with errors = get_flashed_messages(category_filter=["account_err"]) %}
+    {% with errors = get_flashed_messages(category_filter=["message"]) %}
         {% if errors %}
             {%- for msg in errors %}
-                <div class="alert alert-danger">
+                <div class="alert alert-{{ msg.split(';', 1)[0] }}">
                     <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
-                    {{ msg }}
+                    {{ msg.split(';', 1)[1] }}
                 </div>
```

Comment:
We can talk later about ways to avoid splitting strings in the template. It's never best to have a string with a special character (`;`) in order to parse, etc.